### PR TITLE
feat: add migration components

### PR DESCRIPTION
**Pull request by janndriessen**
## **Summary of Changes**

* Adds 2xFLI migration banners
* Adds tailwind prettier plugin

## **Test Data or Screenshots**
Bildschirmfoto 2024-02-14 um 09 56 24

###### _By submitting this pull request, you are confirming the following to be true:_

- I have reviewed the [Contribution Guidelines](https://github.com/IndexCoop/index-app/blob/master/CONTRIBUTING.md).
- I have performed a self-review of my own code.
- I have updated my repository to match the master at `IndexCoop/index-app`.
- I have included test data or screenshots that prove my fix is effective or that my feature works.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,11 +1,7 @@
 ## **Summary of Changes**
 
-&nbsp;
-
 ## **Test Data or Screenshots**
 
-&nbsp;
-
```
Comment (janndriessen) [inline .github/pull_request_template.md:8]: sneaking in this change that I wanted to do for a while

File: .prettierrc.json [hunk 2]
```diff
@@ -2,5 +2,7 @@
   "semi": false,
   "singleQuote": true,
   "jsxSingleQuote": true,
-  "quoteProps": "consistent"
+  "quoteProps": "consistent",
+  "plugins": ["prettier-plugin-tailwindcss"]
```
Comment (0xonramp) [inline .prettierrc.json:6]: 🔥

File: src/app/layout.tsx [hunk 3]
```diff
@@ -27,16 +28,17 @@ export default function RootLayout({ children }: LayoutProps) {
...
               p={[
-                '100px 16px 0px 16px',
-                '100px 16px 0px 16px',
-                '128px 16px 0px 16px',
-                '128px 16px 0px 16px',
+                '48px 16px 0px 16px',
+                '48px 16px 0px 16px',
+                '48px 16px 0px 16px',
+                '48px 16px 0px 16px',
               ]}
```
Comment (0xonramp) [inline src/app/layout.tsx:42]: Could collapse this all into a single `p='48px 16px 0px 16px'` now

**Review by 0xonramp — COMMENTED** [review]
LGTM - one minor nit

File: src/app/layout.tsx (hunk 3 above)
Comment (janndriessen) [reply src/app/layout.tsx:42]: Fixed in https://github.com/IndexCoop/index-app/pull/991/commits/43e89d42ede8743e18f374aae04efd22778e7df2

**Pull request merged**